tests/BUILD

```diff
@@ -267,6 +267,19 @@ rule_test(
     rule = "//tests/cc_haskell_import:cc-bin",
 )
 
+# TODO(Profpatsch) blocked on https://github.com/bazelbuild/bazel/issues/6093
```

Comment:
That was sadly not the whole story.

`bazel run` doesn’t sandbox its builds, so the python script currently succeeds inside `bazel run` but fails when called inside `bazel test`. It could be that the test here needs some additional inputs, or that we are not propagating enough dependencies.

I’ll create a followup issue, because getting it to run from a test (as opposed to manually) is not in scope of this particular PR.